docs/content/deployment/guides/kubernetes/how-to-migrate-your-instance.mdx

```diff
@@ -1,24 +1,42 @@
 ---
-title: How to Migrate | Dagster
+title: "Migrating a Dagster instance while upgrading | Dagster Docs"
 description: We walk through how to migrate your Dagster instance using a Kubernetes Job from the Helm chart.
 ---
 
-# How to Migrate Your Instance
-
-## Overview
+# Migrating a Dagster instance while upgrading
 
 When upgrading your Dagster version, you may also need to migrate your Dagster instance. Migrations will only be required if you are upgrading your minor version.
 
-Here, we walk you through the migration process in a Kubernetes environment.
+In this guide, we'll walk you through the migration process in a Kubernetes environment.
+
+---
+
+## Prerequisites
+
+Before you get started, complete the following:
+
+- **Check the [Dagster migration guide](/migration).** Use the guide to verify any additional steps you may need to take to upgrade your Dagster version.
+- **Back up your PostgreSQL database.**
+
+---
+
+## Step 1: Upgrade the Helm chart
 
-## Walkthrough
+<Note>
+  <strong>Heads up!</strong> This guide assumes only one release of the Dagster
+  Helm chart is installed in your Kubernetes cluster. All commands should be run
+  in the proper namespace.
+</Note>
 
-For simplicity, we assume that you have only installed one release of the Dagster Helm chart in your Kubernetes cluster. All commands should be run in the proper namespace.
+First, run [`helm upgrade`](https://helm.sh/docs/helm/helm_upgrade/) with your desired Dagster chart version and Helm values.
```

Comment:
Example command here?

Reply:
i think this is something like
```
helm repo update
```
and then the same command from the main guide here https://docs.dagster.io/deployment/guides/kubernetes/deploying-with-helm#step-7-install-the-dagster-helm-chart

(making some assumptions about the existing helm release name - the word dagster appears three times in the example command here and i wish i was 100% certain what each of those 'dagster's corresponded to)

```
helm upgrade --install dagster dagster/dagster -f /path/to/values.yaml
```

Reply:
@gibsondan It looks like [`helm repo update`](https://helm.sh/docs/helm/helm_repo_update/) updates the repo's version of Helm - is this correct? Folks need to upgrade Helm and then upgrade the Dagster chart (`helm upgrade`?)